pyproject.toml

```diff
@@ -35,6 +35,9 @@ uvicorn = { version = "^0.22", extras = ["standard"], optional = true }
 # extras: websocket_client
 websocket-client = { version = "^1.7.0", optional = true }
 websockets = { version = "^12.0", optional = true }
+# extras: rest
+requests = { version = "^2.31.0", optional = true }
+types-requests = { version = "^2.31.0", optional = true }
```

Comment:
```suggestion
# extras: rest
requests = { version = "^2.31.0", optional = true }
```
Installing this is handled by mypy when needed. I don't think we should add unneeded packages by default.

Reply:
this is a holdover from me not properly installing with dev,test groups. fixed.